src/audits/performance/user-timings.js

```diff
@@ -48,36 +48,68 @@ class UserTimings extends Audit {
       });
     }
 
-    let timingsCount = 0;
+    let userTimings = [];
+    let navigationStartTime;
+    let measuresStartTimes = {};
+    let traceStart = false;
 
     // Fetch blink.user_timing events from the tracing data
     const timelineModel = new TimelineModel(artifacts.traceContents);
     const modeledTraceData = timelineModel.timelineModel();
-    const key = [...modeledTraceData.mainThreadAsyncEvents().keys()].find(
-      key => key.title === 'User Timing'
-    );
-    let userTimings = modeledTraceData.mainThreadAsyncEvents().get(key);
 
-    // Reduce events to record only useful information
-    if (typeof userTimings !== 'undefined') {
-      timingsCount = userTimings.length;
-      userTimings = userTimings.map(ut => {
-        return {
-          name: ut.name,
-          startTime: ut.startTime,
-          endTime: ut.endTime,
-          duration: ut.duration.toFixed(2) + 'ms',
-          args: ut.args
-        };
-      });
-    }
+    modeledTraceData.mainThreadEvents().filter(
+      // Get all blink.user_timing events
+      ut => ut.hasCategory('blink.user_timing') || ut.name === 'TracingStartedInPage'
+    ).forEach(ut => {
+      if (ut.phase === 'R' || ut.phase === 'I') {
```

Comment:
Can you add a comment on what these phases are? Could add a link to https://docs.google.com/document/d/1CvAClvFfyA5R-PhYUmn5OOQtYMH4h6I0nSsKchNAySU/preview#heading=h.uxpopqvbjezh

(btw ignore what it says about deprecated.. it's wrong. :)